tensorflow/python/keras/integration_test/gradient_checkpoint_test.py

```diff
@@ -75,7 +75,7 @@ def _limit_gpu_memory():
   if gpus:
     tf.config.experimental.set_virtual_device_configuration(
         gpus[0],
-        [tf.config.experimental.VirtualDeviceConfiguration(memory_limit=1152)])
+        [tf.config.experimental.VirtualDeviceConfiguration(memory_limit=1024)])
```

Comment:
Is this an accidental revert of a previous change?

Reply:
that is intentional, as that fix does not seem to work 100% of time.  We are working on debugging it, but in the meantime wanted to remove the previous fix, and put a no_rocm tag on that test instead